__integrations__/index.test.ts

```diff
@@ -505,6 +508,78 @@ describe("Create Advanced Message", () => {
     }
   );
 
+  it("should return the message without Read status, if user is allowed to read it", async () => {
+    const fiscalCode = anAutoFiscalCode;
+    const serviceId = aValidServiceId;
+
+    const body = {
+      message: {
+        fiscal_code: fiscalCode,
+        feature_level_type: "ADVANCED",
+        content: aMessageContent
+      }
+    };
+
+    const nodeFetchWithoutPermission = getNodeFetch({
+      "x-subscription-id": serviceId
+    });
+    const nodeFetch = getNodeFetch({
+      "x-subscription-id": serviceId,
+      "x-user-groups":
+        customHeaders["x-user-groups"] +
+        ",ApiMessageWriteAdvanced,ApiMessageReadAdvanced"
+    });
+
+    const result = await postCreateMessage(nodeFetch)(body);
+
+    expect(result.status).toEqual(201);
+
+    const messageId = ((await result.json()) as CreatedMessage).id;
+    expect(messageId).not.toBeUndefined();
+
+    // Wait the process to complete
+    await delay(WAIT_MS);
+
+    // Check response having `ApiMessageReadAdvanced` authorization
+    const resultGet = await getSentMessage(nodeFetch)(fiscalCode, messageId!);
+
+    expect(resultGet.status).toEqual(200);
+    const detail = (await resultGet.json()) as ExternalMessageResponseWithContent;
+
+    expect(detail).toMatchObject(
+      expect.objectContaining({
+        message: expect.objectContaining({
+          id: messageId,
```

Comment:
what the difference with [here](https://github.com/pagopa/io-functions-services/pull/211/files#diff-11319b95f58a7ce1c5d83c788ebcf5364088fc46175ff6c9443e278a0f9b3759L475)?